exonum-java-binding/package_app.sh

```diff
@@ -27,18 +27,29 @@ function build-exonum-java() {
       -Drust.libraryPath="${RUST_LIBRARY_PATH}"
 }
 
-function build-exonum-java-macos() {
-    export RUSTFLAGS="-C link-arg=-Wl,-rpath,@executable_path/lib/native"
+# Prepares native environment for building with build-exonum-java function. The first argument of function is platform
+# dependent rpath value for linker that points to the directory where code is executed/loaded. For currently supported
+# architectures they are "@loader_path" for OSX and "$ORIGIN" for Linux. The second argument is full name of the
+# java_binding library for current platform.
+function build-exonum-java-for-platform() {
+    # This will point to the directory with native libraries in case code was executed by "exonum-java"
+    local path_lib_from_exe="$1/lib/native"
+    # This will point to the directory with native libraries in case code was loaded by java
+    local path_lib_from_lib=$1
```

Comment:
Is it like current directory?

Reply:
Yes. This one is used to load `libstd-XXX` when `libjava_bindings` is loaded with java.